src/styles/styles.js

```diff
@@ -1880,6 +1880,12 @@ const styles = {
         width: '150%',
     },
 
+    fullscreenCardTablet: {
```

Comment:
It seems like the "tablet" styles are triggered with `isMediumScreenWidth`. I would suggest we align the words to be consistent here and just use something like `fullscreenCardMediumScreen`

Reply:
Changed all
- workspaceCardTablet :arrow_right: `workspaceCardMediumScreen`
- fullscreenCardTablet :arrow_right: `fullscreenCardMediumScreen`
- workspaceCardContentTablet :arrow_right: `workspaceCardContentMediumScreen`